lib/archethic/contracts/interpreter/condition.ex

```diff
@@ -263,6 +263,14 @@ defmodule Archethic.Contracts.ConditionInterpreter do
     {node, acc}
   end
 
+  # Whitelist the get_first_transaction_address/0 function in condition
+  defp prewalk(
+         node = {{:atom, "get_first_transaction_address"}, _, []},
```

Comment:
No need for that clause because you added it to utils.prewalk which is the last clause already

Reply:
there are d/f clauses I think